hack/release-tool.py

```diff
@@ -0,0 +1,129 @@
+
+                def update_annotations(obj):
+                    message = f'Manually {action}ed per TRT'
+                    if custom_message is not None:
+                        message = custom_message
+
+                    for annotations in (obj.model.image.metadata.annotations, obj.model.metadata.annotations, obj.model.tag.annotations):
+                        if action == 'accept':
+                            annotations['release.openshift.io/phase'] = 'Accepted'
+                        elif action == 'reject':
+                            annotations['release.openshift.io/phase'] = 'Rejected'
+                        else:
+                            raise ValueError(f'Unsupported action specified: {action}')
+
+                        annotations['release.openshift.io/message'] = message
+
+                        if custom_reason is not None:
+                            annotations['release.openshift.io/reason'] = custom_reason
+
+                    logger.info(f'{action.capitalize()}ing: {ns}/{name}:{release}')
+                    if execute:
+                        logger.debug(f'Updated payload:\n{json.dumps(obj.model._primitive(), indent=4)}')
+                        return True
+                    else:
+                        logger.info(f'Updated payload:\n{json.dumps(obj.model._primitive(), indent=4)}')
+                        logger.warning('You must specify "--execute" to permanently apply these changes')
+                        exit(0)
+
+                update_annotations(tag)
+                tag.replace()
+                logger.info(f'Release {release} updated successfully')
+                logger.info(f'Backup written to: {backup_filename}')
+
+        except (ValueError, OpenShiftPythonException, Exception) as e:
+            logger.error(f'Unable to update release: "{release}"')
+            raise e
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Manually accept or reject release payloads')
+    parser.add_argument('-m', '--message', help='Specifies a custom message to include with the update', default=None)
+    parser.add_argument('-r', '--reason', help='Specifies a custom reason to include with the update', default=None)
+    parser.add_argument('--execute', help='Specify to persist changes on the cluster', action='store_true')
+    parser.add_argument('--admin', help='Perform operations as "system:admin"', action='store_true')
+
+    config_group = parser.add_argument_group('Configuration Options')
+    config_group.add_argument('-v', '--verbose', help='Enable verbose output', action='store_true')
+
+    ocp_group = parser.add_argument_group('Openshift Configuration Options')
+    ocp_group.add_argument('-c', '--context', help='The OC context to use (default is "app.ci")', default='app.ci')
+    ocp_group.add_argument('-n', '--namespace', help='The namespace prefix to use (default is "ocp")', default='ocp')
```

Comment:
Is this really a root option? Or are the namespaces derived from the product (OCP, OKD...) and other data?

Reply:
Changed to `--name` with supported options of `ocp` or `okd` and updated logic to handle `okd` properly.